src/utils/index.test.js

```diff
@@ -57,3 +65,36 @@ it('isStale', () => {
   expect(isStale(incomingResource, state)).toBe(true);
   expect(isStale(existingResource, state)).toBe(false);
 });
+
+it('fetchLogs', () => {
```

Comment:
This test is essentially testing itself, it essentially goes like this:
1. define mock function
1. spyOn API
1. call mock function
1. expect mock function to have been called

Instead we really want to test the actual `fetchLogs` function by calling it (not a mocked version of it). Something like this:

```js
it('fetchLogs', () => {
  const stepName = 'kubectl-apply';
  const stepStatus = { container: 'step-kubectl-apply' };
  const taskRun = { pod: 'pipeline-run-123456', namespace: 'default' };

  const logs = 'fake logs';
  jest.spyOn(API, 'getPodLog').mockImplementation(() => logs);

  const returnedLogs = fetchLogs(stepName, stepStatus, taskRun);
  expect(API.getPodLog).toHaveBeenCalledWith(expect.objectContaining({
    container: stepStatus.container,
    name: taskRun.pod,
    namespace: taskRun.namespace
  }));
});
```